prac_05/emails.py

```diff
@@ -1,5 +1,32 @@
 """
 CP1404/CP5632 Practical
-Emails
+Emails in dictionaries
 """
 
+
+def main():
+    """ Get email and name from user and store in a dictionary"""
+    email_dict = {}
+
+    email = input("Email: ")
+    while email:
```

Comment:
while email is... ??? I would put while email != "":

Reply:
Cheers for the review Matt.
`while email:` does the same thing as `while email != ""`, just shorthand. I'm trying to ask Lindsay what his thoughts are on this shorthand.

Reply:
Either way is OK, but I personally find `while email != ""` easier to read. This may just be my exposure, experience, so I don't have a strong view. I believe the practice of using sequences as True/False is common and accepted.